src/components/Profile/Profile.js

```diff
@@ -0,0 +1,49 @@
+import React, { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+// import { Spinner } from 'react-spinners-css';
```

Comment:
- [ ] Kindly remove all commented line code, It is always a good practice to keep your code clean from such comments. 😉 👍🏻